text/dependencies/rfcs/0003-dependency-management-overview.md

```diff
@@ -121,7 +121,7 @@ many different OS compatibilities to be used in the buildpack.
 In order to implement these features in the buildpacks, there are three main
 phases:
 
-1. Federated Model Adoption In Buildpacks: [Phase 1 RFC](https://github.com/paketo-buildpacks/rfcs/blob/dependency-management-top-level/text/dependencies/rfcs/0000-dependency-management-overview.md)
+1. Federated Model Adoption In Buildpacks: [Phase 1 RFC](https://github.com/paketo-buildpacks/rfcs/blob/main/text/dependencies/rfcs/0004-dependency-management-phase-one.md)
```

Comment:
You can always use relative paths for these links so that you don't have to mention the branch name at all.

```suggestion
1. Federated Model Adoption In Buildpacks: [Phase 1 RFC](./text/dependencies/rfcs/0004-dependency-management-phase-one.md)
```